main.css

```diff
@@ -101,7 +101,53 @@ body {
 .nav-category a {
   color: var(--primary-color);
 }
+/* card */
+.card-info {
+  margin: 0;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: var(--black);
+  color: var(--primary-color);
+  font-size: 1rem;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+.card-title {
+  font-size: 1.2rem;
+}
+.card-display-container {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-row-gap: 2rem;
+  grid-column-gap: 5rem;
+}
+.grid-container {
+  width: 85%;
+  margin: 2rem auto;
+}
 
+/* section feautured */
+.featured-cards {
+  margin: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 2rem;
+}
+.section-title {
+  color: var(--accent-color);
+  text-align: center;
+  font-size: 2rem;
+  font-weight: 900;
+  margin: 1rem;
+}
+.offer-info {
+  color: greenyellow;
```

Comment:
Try to use variables for colors throughout the code for consistency.

Reply:
Ok